index.html

```diff
@@ -554,16 +554,19 @@ <h2>
 &lt;/button&gt;
 </pre>
       <p>
-        This example waits to be fullscreen before locking the screen
-        orientation and starting.
+        This example waits for fullscreen, then locks to landscape before
+        calling <code>ready()</code>.
       </p>
       <pre class='example html'>
 &lt;script&gt;
-  var start = function() {
-    document.onfullscreenchange = function() {
-      screen.orientation.lock('natural').then(startInternal);
-    }
-    document.documentElement.requestFullscreen();
+  function ready(){
+    console.log("Now full screen and ready to start, locked to " + screen.orientation.type);
```

Comment:
This line is getting a little long, which means that anyone looking at this on a phone will see: 

![screenshot 2019-01-17 21 36 40](https://user-images.githubusercontent.com/870154/51312748-19b6a300-1aa0-11e9-90e3-741b1bff9511.png)

It might be good to:

```JS
  const { type } = screen.orientation;
  console.log("Fullscreen and locked to ${type}. Ready!");
```

Reply:
(feel free to adjust the wording)